debezium-connector-sqlserver/src/test/resources/META-INF/services/io.debezium.pipeline.notification.channels.NotificationChannel

```diff
@@ -0,0 +1,2 @@
+io.debezium.pipeline.notification.channels.SinkNotificationChannel
```

Comment:
@mfvitale This can be probably also removed

Reply:
Are you refer to the file or the specific implementation?